docs/eventing/brokers/broker-types/mt-channel-based-broker/README.md

```diff
@@ -0,0 +1,117 @@
+# Multi-tenant channel based Broker
+
+The Multi-tenant channel based Broker (MTChannelBasedBroker) uses [Channels](../../../channels) for event routing. It is shipped by default with Knative Eventing.
+
```

Comment:
```suggestion
The Multi-tenant channel based Broker (MTChannelBasedBroker) uses [Channels](../../../channels) for event routing. It is shipped by default with Knative Eventing.
Users should prefer native Broker implementations over the MTChannelBasedBroker and channel combo because it is usually more efficient. 
```